docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -189,7 +189,11 @@ In addition to the rules above, the default mapping can contain the optional
 
 If no mapping entry matches and no default is configured, the access is denied.
 
-Example JSON configuration file:
+The configuration JSON can either be retrieved from a file specified via
+``hive.s3.security-mapping.config-file`` or from a REST-endpoint specified via
+``hive.s3.security-mapping.config-uri``.
```

Comment:
Can we make `hive.s3.security-mapping.config-file` to accept URI? Then you don't need to have two properties to maintain. Also the way we read JSON is quite reused (for example for file based access controls) so if your change could be generic enough other places could also benefit from that.

Notice that we already are doing this where one can give an URI for `http-server.authentication.jwt.key-file`.

Reply:
My apologies, I've been very busy and haven't followed up here. I appreciate your patience on this. Let me try updating it to use the existing config and get this merged. I do like the idea of a single config.